### PR TITLE
Add GPU support for draw_segmentation_masks

### PR DESCRIPTION
**Pull request by mshr-h**
#7683

### CODE REVIEW

**Review by NicolasHug — COMMENTED** [review]
Thanks for the PR @mshr-h . We should probably add a quick test to make sure it works as expected, by e.g. adding the `cpu_and_gpu()` decorator to this test: https://github.com/pytorch/vision/blob/fc83b280387d92fe6aea73f35c1ed4ad1bb739da/test/test_utils.py#L206

See https://github.com/pytorch/vision/blob/fc83b280387d92fe6aea73f35c1ed4ad1bb739da/test/test_functional_tensor.py#L542-L543 for an example of how it's used

**Comment by mshr-h** [comment]
Thank you for your quick review.
I updated the unit test.

I need to ask my employer to sign the CLA. I guess it takes a few weeks.

**Review by pmeier — APPROVED** [review]
CI failures are unrelated. Thanks a lot @mshr-h!

> I need to ask my employer to sign the CLA. I guess it takes a few weeks.

Unfortunately, we cannot merge until that is done. We should get a notification as soon as that is done, but in case we miss it, feel free to ping us here.

**Comment by mshr-h** [comment]
@pmeier 
I've completed signing the CLA.

File: test/test_utils.py
Comment (NicolasHug) [inline test/test_utils.py:250]: I'll leave this here to avoid churn but we typically don't need to run those tests on cuda, as there's nothing cuda-spcecific there. Same for `test_draw_no_segmention_mask`

**Review by NicolasHug — APPROVED** [review]
Thanks @mshr-h , I fixed a conflict issue and I'll merge when green

**Pull request merged**
